src/Serilog.Sinks.AzureDataExplorer.Tests/AzureDataExplorerSinkOptionsExtensionsTests.cs

```diff
@@ -0,0 +1,137 @@
+using System.Security.Cryptography.X509Certificates;
+using System.Security.Cryptography;
+using Serilog.Sinks.AzureDataExplorer.Extensions;
+
+namespace Serilog.Sinks.AzureDataExplorer.Tests;
+
+public class AzureDataExplorerSinkOptionsExtensionsTests
```

Comment:
Very similar to DurableSink tests , collapse them if you can. Code duplication can be avoided.